lib/matplotlib/scale.py

```diff
@@ -669,19 +657,22 @@ class LogitScale(ScaleBase):
 
     def __init__(self, axis, nonpos='mask'):
         """
-        *nonpos*: {'mask', 'clip'}
-          values beyond ]0, 1[ can be masked as invalid, or clipped to a number
-          very close to 0 or 1
+        Parameters
+        ----------
+        axis : `matplotlib.axis.Axis`
+            Currently unused.
+        nonpos : {'mask', 'clip'}
+            Determines the behavior for values beyond ]0, 1[. They can either
```

Comment:
I think another PR suggested that we should explicitly say "open interval" or "closed interval" for these sorts of things.